### ISSUE TITLE
Can't install on new nette

### ISSUE THREAD

**Issue by xsajto** [wavevision/namespace-translator#7]
Can you fix it please :)

```./composer.json has been updated
Running composer update wavevision/namespace-translator
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires wavevision/namespace-translator 1.0.6 -> satisfiable by wavevision/namespace-translator[1.0.6].
    - wavevision/namespace-translator 1.0.6 requires nette/application 3.0.8 -> found nette/application[v3.0.8] but it conflicts with your root composer.json require (3.1.2).

Use the option --with-all-dependencies (-W) to allow upgrades, downgrades and removals for packages currently locked to specific versions.

Installation failed, reverting ./composer.json and ./composer.lock to their original content.```

**Comment by rozsival** [comment]
Hey @sedlakos,

I'm sorry, but this is not currently possible, since `nette/application:3.1` introduced some breaking changes that need to be taken in count.

We're planning to upgrade our libraries to support `v3.1` but it's not a priority for us at the moment. If you need the support, your help in form of a PR will be gratefully welcome!

**Comment by rozsival** [comment]
Hey @sedlakos,

this has been resolved in `v1.0.7` 🚀.
